File: EventServiceApi/EventServiceApi/Models/Cemetery.cs
using EventServiceApi.ViewModels;
using System.ComponentModel.DataAnnotations;

namespace EventServiceApi.Models
{
    public class Cemetery
    {
        [Key]
        public long Id { get; set; }

        [Required]
        [MaxLength(50)]
        public string Name { get; set; }

        [Required]
        [MaxLength(50)]
        public string RequiredPhoneNumber { get; set; }

        [MaxLength(50)]
        public string AdditionalPhoneNumber { get; set; }

        [Required]
        public string City { get; set; }

        public string District { get; set; }

        public string Email { get; set; }

        public string Website { get; set; }

        public CemeteryViewModel ConvertToViewModel()
        {
            return new CemeteryViewModel()
            {
                CemeteryId = Id,
                CemeteryName = Name,
                CemeteryRequiredPhoneNumber = RequiredPhoneNumber,
                CemeteryAdditionalPhoneNumber = AdditionalPhoneNumber,
                CemeteryCity = City,
                CemeteryDistrict = District
            };
        }
    }
}

File: app/src/main/java/com/example/tadbir/Cemetery.java
package com.example.tadbir;

import com.google.gson.annotations.SerializedName;

public class Cemetery {
    @SerializedName("Id")
    public long id;
    @SerializedName("Name")
    public String firstName;
    @SerializedName("RequiredPhoneNumber")
    public String phoneNumber;
    @SerializedName("AdditionalPhoneNumber")
    public String additionalPhoneNumber;
    @SerializedName("City")
    public String city;
    @SerializedName("District")
    public String district;
    @SerializedName("Email")
    public String email;
    @SerializedName("Website")
    public String website;
}
